pyiron_atomistics/lammps/base.py

```diff
@@ -60,8 +64,25 @@ class DumpData:
     positions: List = field(default_factory=lambda: [])
     computes: Dict = field(default_factory=lambda: {})
 
+class LammpsStaticOutput(StaticWithPressureOutput):
+
+    def __init__(self, job: "LammpsBase"):
```

Comment:
Doesn't this resolve OK with the actual class if you have `from __future__ import annotations`?

Reply:
Didn't know about it, but reading it, yeah, it should.